src/components/UI/AppDetails/VersionsRow.js

```diff
@@ -0,0 +1,87 @@
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import Copyable from 'shared/Copyable';
+import Truncated from 'UI/Truncated';
+
+const INITIAL_MAX_CHART_VERSIONS = 2;
+
+const ChartVersion = styled(Copyable)`
+  margin-bottom: 10px;
+  float: left;
+`;
+
+const AppVersion = styled(Copyable)`
+  display: inline-block;
+  float: none;
+  position: relative;
+  left: 8px;
+  code {
+    background-color: ${(props) => props.theme.colors.darkBlueLighter8};
+    color: ${(props) => props.theme.colors.darkBlue};
+  }
+`;
+
+const ExpandVersions = styled.button`
+  padding: 0px;
+  background-color: inherit;
+  font-size: 14px;
+  color: ${(props) => props.theme.colors.darkBlueLighter6};
+
+  &:hover {
+    background-color: inherit;
+  }
+`;
+
+const VersionsRow = ({ appVersion, entries, className }) => {
+  const [expandVersions, setExpandVersions] = useState(
+    INITIAL_MAX_CHART_VERSIONS
+  );
+
+  const expand = () => {
+    setExpandVersions(entries.length);
+  };
+
+  return (
+    <tr className={className}>
+      <td>
+        <div>
+          {entries.slice(0, expandVersions).map((appVersionObject) => {
+            return (
+              <ChartVersion
+                key={appVersionObject.version}
+                copyText={appVersionObject.version}
+              >
+                <Truncated numStart={12} as='code'>
+                  {appVersionObject.version}
+                </Truncated>
+              </ChartVersion>
+            );
+          })}
+        </div>
+        <br style={{ clear: 'both' }} />
```

Comment:
Could you move this `br` into the conditional or even better remove it entirely? Or do you intend to have the `+ N versions` button on a new line? I think it should be possible then to give it 100% width to achieve the same effect.

Reply:
I made some changes in general to improve the alignment and allow it fit on the same line

Reply:
<img width="308" alt="Screenshot 2020-04-20 at 3 14 17 PM" src="https://user-images.githubusercontent.com/455309/79724192-b1491800-8319-11ea-9565-c85759489c3b.png">